src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { normalize, parse } from './compose';
+import { ValidationError } from './errors';
+import { DEFAULT_SCHEMA_VERSION, SchemaVersion } from './schemas';
+import { BuildConfig, Composition, ImageDescriptor, Network, Service, Volume } from './types';
+
+export {
+	normalize,
+	parse,
+	BuildConfig,
+	Composition,
+	DEFAULT_SCHEMA_VERSION,
+	ImageDescriptor,
+	Network,
+	SchemaVersion,
+	Service,
+	ValidationError,
+	Volume,
+};
```

Comment:
You'll also need to export `InternalInconsistencyError` if we want to catch it downstream.

Reply:
I think it's better to not handle this error -- it's not even supposed to happen and it's only used to silence the compiler while working with potentially undefined values -- and a global catch handler will catch it anyway. If there's some obscure reason to handle this error, it can be imported with `import { InternalInconsistencyError } from 'docker-compose-parse/errors';`

Reply:
Fair enough, I believe we can use `error.name` in the global handler anyway (and therefore sentry) to find out the type of error.